### PR TITLE
Display transmitted messages in MQTT debug info dialog

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Display transmitted messages in MQTT debug info dialog

Depends on core PR https://github.com/home-assistant/core/pull/65550

Screenshot:
image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/devices/device-detail/integration-elements/mqtt/mqtt-messages.ts [hunk 1]
```diff
@@ -26,6 +25,8 @@ class MQTTMessages extends LitElement {
 
   @state() private _showTopic = false;
 
+  direction: string = "<direction>"
```
Comment (emontnemery) [inline src/panels/config/devices/device-detail/integration-elements/mqtt/mqtt-messages.ts:28]: Is this the right way to do it? Can a class attribute be abstract, so something like:
```ts
abstract readonly direction: string;
```

**Comment by emontnemery** [comment]
The backend change is merged, this should be GTG

File: src/panels/config/devices/device-detail/integration-elements/mqtt/mqtt-messages.ts [hunk 2]
```diff
@@ -50,7 +51,7 @@ class MQTTMessages extends LitElement {
                 (message) => html`
                   <li class="message">
                     <div class="time">
-                      Received
+                      ${this.direction}
```
Comment (bramkragten) [inline src/panels/config/devices/device-detail/integration-elements/mqtt/mqtt-messages.ts:54]: You could have kept it 1 element and add a `direction` property to it 😄

File: src/translations/en.json [hunk 3]
```diff
@@ -885,7 +885,8 @@
         "no_entities": "No entities",
         "no_triggers": "No triggers",
         "payload_display": "Payload display",
-        "recent_messages": "{n} most recently received message(s)",
+        "recent_rx_messages": "{n} most recently received message(s)",
```
Comment (bramkragten) [inline src/translations/en.json:888]: This means we lose all translations for this string

File: src/panels/config/devices/device-detail/integration-elements/mqtt/mqtt-messages.ts (hunk 2 above)
Comment (emontnemery) [reply src/panels/config/devices/device-detail/integration-elements/mqtt/mqtt-messages.ts:55]: Fixed :)

**Review by bramkragten — APPROVED** [review]

**Pull request merged**